package.json

```diff
@@ -4,54 +4,78 @@
   "description": "Skeleton for Express APIs.",
   "main": "app.js",
   "scripts": {
-    "start": "./node_modules/.bin/gulp run",
+    "start": "forever dist/app.js",
     "lint": "./node_modules/.bin/gulp lint",
-    "test": "./node_modules/.bin/gulp test"
+    "test": "./node_modules/.bin/gulp test",
+    "typecheck": "./node_modules/.bin/gulp typecheck",
+    "babel": "./node_modules/.bin/gulp babel"
   },
   "pre-commit": [
-    "lint"
+    "lint",
+    "typecheck",
+    "babel"
   ],
   "repository": {
     "type": "git",
     "url": "git+https://github.com/osu-mist/express-api-skeleton.git"
   },
+  "engines": {
+    "node": "^10.13"
+  },
   "author": "osu-mist",
   "license": "AGPL-3.0",
   "bugs": {
     "url": "https://github.com/osu-mist/express-api-skeleton/issues"
   },
   "homepage": "https://github.com/osu-mist/express-api-skeleton#readme",
   "dependencies": {
+    "@babel/runtime": "^7.4.3",
     "app-root-path": "^2.1.0",
     "body-parser": "^1.18.3",
     "compose-middleware": "^5.0.1",
     "config": "^2.0.1",
     "deasync": "^0.1.14",
     "decode-uri-component": "^0.2.0",
-    "eslint-plugin-jsdoc": "^15.6.1",
     "express": "^4.16.3",
-    "express-basic-auth": "1.1.7",
+    "express-basic-auth": "^1.1.7",
     "express-openapi": "^4.6.5",
     "express-winston": "^3.2.1",
+    "forever": "^1.0.0",
+    "gulp": "^4.0.2",
     "js-yaml": "^3.13.1",
     "jsonapi-serializer": "^3.5.6",
     "lodash": "^4.17.13",
     "moment": "^2.22.2",
     "query-string": "^6.2.0",
     "simple-git": "^1.96.0",
+    "source-map-support": "^0.5.12",
     "swagger-parser": "^6.0.3",
     "winston": "^3.2.1",
     "winston-daily-rotate-file": "^3.10.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.4.3",
+    "@babel/plugin-proposal-optional-chaining": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.4.3",
+    "@babel/preset-env": "^7.4.3",
+    "@babel/preset-flow": "^7.0.0",
+    "babel-eslint": "^10.0.1",
+    "babel-plugin-module-resolver": "^3.2.0",
     "chai": "^4.1.2",
+    "del": "^4.1.0",
     "eslint": "^5.6.0",
     "eslint-config-airbnb-base": "^13.1.0",
+    "eslint-import-resolver-babel-module": "^5.0.1",
+    "eslint-plugin-flowtype": "^3.6.0",
+    "eslint-plugin-jsdoc": "^15.6.1",
     "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-import-order-alphabetical": "^0.0.2",
     "forever-monitor": "^1.7.1",
-    "gulp": "4.0.0",
+    "flow-bin": "^0.96.0",
+    "gulp-babel": "^8.0.0",
     "gulp-eslint": "^5.0.0",
     "gulp-mocha": "^6.0.0",
+    "gulp-sourcemaps": "^2.6.5",
     "pre-commit": "^1.2.2"
   }
 }
```

Comment:
It looks like the order of the plugins is off slightly. Not a big deal, but you could fix this by doing `yarn remove <something>` and then `yarn add <something>` for some package in dependencies. As long as this doesn't modify any versions. You could also do `npx sort-package-json`.